lib/tests/it/main.rs

```diff
@@ -961,6 +961,26 @@ async fn test_container_import_export_v1() {
         .unwrap();
 }
 
+
+///Test layers
+#[tokio::test]
+async fn test_layers() -> Result<()> {
+    let fixture = Fixture::new_v1()?;
+    let srcpath = fixture.path.join("src/repo");
+    let srcrepo_parsed = ostree_ext::cli::parse_repo(&srcpath).unwrap();
+    let ir = ostree_ext::cli::parse_imgref("ostree-remote-image:containers-storage:quay.io/fedora/fedora-coreos:testing-devel")
```

Comment:
This string won't work; I think you mean e.g. `ostree-unverified-image:containers-storage:quay.io/fedora/fedora-coreos:testing-devel`.  But even then, I don't think we can do this in a unit test.  A general idea is that unit tests shouldn't change any global system state, and pulling a container image into e.g. `~/.local/share/containers` is global system state.

We could make that type of test opt in?  Alternatively we have e.g. `integrationtest.rs` - could make sense there.